.config/cargo_spellcheck.dic

```diff
@@ -22,19 +22,17 @@ spreaded
 callee
 pre
 defrag
-defragment/SD
 defragmentation
 defragmentation
 growable
 codegen
-endian/P
-deallocate/X
+deallocate
+deallocation
```

Comment:
@drahnr I had to revert your suggestion of `deallocate/X`, didn't work unfortunately.

Reply:
Odd, might be related to drahnr/cargo-spellcheck#159 ( talked to @Robbepop about this a couple of days(?)) ago. Pending impl